pom.xml

```diff
@@ -628,12 +628,6 @@
   <dependencyManagement>
     <dependencies>
       <!-- Scala -->
-      <dependency>
-        <groupId>org.scala-lang</groupId>
-        <artifactId>scala-library</artifactId>
-        <version>${scala.version}</version>
-      </dependency>
```

Comment:
The scala-library in root pom affects the dependency of all the modules, while conflicts with hudi-flink, so remove it.